src/Engine/Entity/Entity.cpp

```diff
@@ -336,6 +334,9 @@ unsigned int Entity::GetUniqueIdentifier() const {
     return uniqueIdentifier;
 }
 
+void Entity::SetDirection(glm::vec3 rot) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'SetDirection' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)